### ISSUE TITLE
Add Filipino lang

### ISSUE THREAD

**Issue by JohnHenryGaspay** [tesseract-ocr/langdata#84]
Would it be also good if you guys can support filipino language here in the Philippines.

**Comment by JohnHenryGaspay** [comment]
@theraysmith I've noticed that there is no Filipino language on the list of data.

**Comment by amitdo** [comment]
https://github.com/tesseract-ocr/tessdata/raw/master/best/fil.traineddata

**Comment by theraysmith** [comment]
My training text corpus does not distinguish between fil and tgl, while they show up in ISO-639-2T as distinct. For some reason that I can't remember now, the language code has switched from tgl to fil in the "best" models that I pushed recently.

Does the fil language do what you want?
If not please try to explain why.
You could also try Latin, which attempts to cover *all* latin-based languages.

**Comment by JohnHenryGaspay** [comment]
@amitdo I've tried adding it to the language folders but when selecting fil as language the app always shut down.

**Comment by JohnHenryGaspay** [comment]
@theraysmith Yes our national language here in the Philippines is Filipino(fil) and tagalog(tgl) is the old name for that. I've tried the Latin but it's not working.

**Comment by Shreeshrii** [comment]
I tested just now, with both best/fil and tgl (4.00.00alpha traineddatas) and they work with tesseract built from latest github code.

```
 tesseract fil-test.png fil-test-best-fil --oem 1 --psm 6 -l best/fil --tessdata-dir ../

 tesseract fil-test.png fil-test-tgl --oem 1 --psm 6 -l tgl  --tessdata-dir ../

```
Files attached. To me best/fil seems more accurate.
I took a snapshot from tgl wikipedia page.

[fil-test-tgl.txt](https://github.com/tesseract-ocr/langdata/files/1233974/fil-test-tgl.txt)
[fil-test-best-fil.txt](https://github.com/tesseract-ocr/langdata/files/1233973/fil-test-best-fil.txt)
![fil-test](https://user-images.githubusercontent.com/5095331/29453594-cc44fcd0-8427-11e7-9fb1-1b97d70283b3.png)

**Comment by amitdo** [comment]
>I've tried adding it to the language folders but when selecting fil as language the app always shut down.

You should try running Tesseract from the command-line.
